machine-learning-notebooks/DGA_Detection_ManagedIdentity.ipynb

```diff
@@ -0,0 +1,577 @@
+{
```

Comment:
<p>You mention visualizations at the beginning of the notebook but I don't see any.</p><p> </p><p>Also here it would be worth spelling out exactly what the results are in the output CSV. </p><p>I think I can work it out but it's not super transparent. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/276/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>